### PR TITLE
Use sequelize's bundled copy of bluebird if it is available

### PR DESCRIPTION
**Pull request by schmod**
Sequelize's [Managed Transactions](http://docs.sequelizejs.com/manual/tutorial/transactions.html) often do not work correctly in conjunction with sequelize-fixtures.  

For instance, this example shows a case where a managed transaction works correctly, as well as a case where sequelize-fixtures causes the transaction's "context" to be lost:

```js
sequelize.transaction(function () {
  models.person.findAll();  // will automatically use the transaction 
  return sequelize_fixtures.loadFile('fixtures/test_data.json', models).then(function(){
        return models.person.findAll(); // will NOT automatically use the transaction
  });
});
```

---

sequelize-fixtures depends on Bluebird 2.x, while most recent versions of Sequelize depend on Bluebird 3.x.  As a result, npm will install two different copies of Bluebird, and Sequelize and sequelize-fixtures will end up using completely different copies of Bluebird.

The API of Bluebird 2.x is similar enough to 3.x that most users aren't going to care that sequelize-fixtures is using an older version.  However, Sequelize's automatic/managed transaction functionality utilizes [CLS](https://github.com/othiym23/node-continuation-local-storage#readme), which can only be used with Bluebird if a special [patch](https://github.com/TimBeyer/cls-bluebird) is applied to the Bluebird instance.

While Sequelize patches the instance of bluebird that it uses internally, the version used by sequelize-fixtures is not patched.

tl;dr; We can sidestep this entire issue (while also avoiding any huge compatibility breaks for existing users) by grabbing Sequelize's copy of bluebird instead of providing our own.

### CODE REVIEW

**Comment by schmod** [comment]
Any chance of getting this reviewed/merged?
